apply-templates.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+[ -f versions.json ] # run "versions.sh" first
```

Comment:
Since `versions.sh` doesn't exist, maybe we remove the command but replace it with one that's just ensuring the needed versions file exists first.

Reply:
Done.